pallets/pool-system/src/impls.rs

```diff
@@ -482,8 +482,6 @@ mod benchmarks_utils {
 				pool.parameters.max_nav_age = 999_999_999_999;
 			});
 
-			T::NAV::update_nav(pool_id).unwrap();
-
```

Comment:
A non existing nav should be none.

Reply:
There is even an issue for that IIrc

Reply:
Or an dispatch error if the pool does not have assets. Mmh. Can we actually check that?

Reply:
Nevermind. We will not be able to check that on the loan side. But it feels a little sketchy.

Reply:
In the previous version, an `update_nav` was mandatory because it set the nav value that later in `close_epoch` read by `nav()`. Now, as it is stored, getting the nav over an empty or non-existent pool will return 0. 

I can return a `None` if the pool does not exist if it's preferred, although it adds a new error you need to deal with.

I feel these methods are strange for belonging to the loan. Why not adding a new method to `PoolMutate` trait that allows setting the portfolio valuation in the pool? instead of being the pool that fetches the nav? I feel like the portfolio valuation value belongs to the pool.

Reply:
> There is even an issue for that IIrc

Can you reference this?

Reply:
I like the idea. But would not add to poolmutate but rather a new trait. Just to not expose it to the registry. Maybe some consumer like logic. Something like valuation updated and then the consumer can do with it what it want.

Reply:
At the same time it somehow does belong to loans. I am undecided

Reply:
Can't find the ticket

Reply:
I feel like the action belongs to the loans, but the storage to the pool. So the entity in charge of dispatching the `update_nav` should be the loan and then stored in the pool. At least this is how I see it.

Reply:
By now, is it ok returning 0, or do you prefer `None` for existing pools that never have called `update_nav` before?